pages/common/vdir.md

```diff
@@ -0,0 +1,32 @@
+# vdir
+
+> List directory contents.
+> Drop-in replacement for `ls -l`.
+
+- List files in the current directory, one per line, with details:
```

Comment:
So this doesn't list subdirectories?

Reply:
Nope, it doesn't. It is a drop-in replacement for `ls -l` as I said. See: 

![image](https://user-images.githubusercontent.com/14198070/52825856-b596f700-30be-11e9-9009-efff1b741899.png)

Reply:
Ah, by "subdirectories" I meant whether it listed directories that are subdirectories fot he directory that you're listing the contents of.  By that screenshot it looks like it does.

Reply:
@sbrl Oh, yeah, it does! Do you think examples would be better rewritten saying "files and directories" instead of just "files" everywhere?

Reply:
Ah, I see. I think it would probably be clearer if we did use `files and directories` in this instance.

Reply:
You're right, pushed a reword.